### DISCUSSION TITLE
Installed in Homebridge, nothing shows up in Home?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [RaresAil/homebridge-levoit-air-purifier#36] [Q&A] [answered]
I installed this, followed the directions... my VeSync credentials are correct... So how do you get it to show up in Apple Home? All the other Homebridge plugins I use, the devices appeared almost immediately. But this one, there's nothing. Anyone get this to work? (I asked on Reddit as well, and someone else couldn't figure it out either)

**Comment by SawkeeReemo** [comment, marked as answer]
Answered my own question not 10 seconds after I asked it... after trying to set it up only on my phone for a week. 🤦‍♂️ 

To anyone searching: After you follow the directions here to set up, go to the plugin page on Homebridge, and you'll see a little QR code symbol in the bottom lower right of the plugin itself. Click that, and you'll get the Apple HomeKit QR code to add it as an accessory.

I'm gonna go slap myself for a while. Sorry to bother! And thanks for the plugin!

**Reply by RaresAil** [reply]
Hey, no worries, there is something in programming called "Rubber Duck Debugging" is basically when you have an issue that you don't know how to fix, try to speak it out load and it will make more sense 😄😄

**Reply by SawkeeReemo** [reply]
Ha! I love it. Although… mine was more of a PEBKAC error. (Problem. Exists. Between. Keyboard. And. Chair.) 🤦‍♂️ 🤣

**Comment by DejanOnTheGo** [comment]
Hey thank you for the hint. But in HOOBS I dont see an addition QR code?
Any additional hints?

**Reply by SawkeeReemo** [reply]
Sorry, I just googled what HOOBS is, but I've never seen that before. Not sure how that works with all this. hopefully someone reading can clue you in.

**Reply by RaresAil** [reply]
Hey, @DejanOnTheGo the QR code is in the hombridge web app in the settings section of the plugin

**Comment by tomtom00700** [comment]
the plugin give me the information - login/password not right... stupid, on the phone is the login ok... (Copy/paste)

**Reply by tomtom00700** [reply]
[7/9/2025, 4:12:04 PM] [asf] [DEBUG]: [LOGIN] The authentication failed!! JSON: {"traceId":"1752077523586","code":-11202022,"msg":"the account does not exist","module":null,"stacktrace":null,"result":null}
